community/security/src/main/java/org/neo4j/server/security/auth/AuthProcedures.java

```diff
@@ -34,7 +34,14 @@
     @Context
     public AuthSubject authSubject;
 
+    @Deprecated
     @Procedure( name = "dbms.changePassword", mode = DBMS )
```

Comment:
Support for deprecatedBy is in another PR, so we need to add that once the other PR is in (probably a small followup PR with a one-line addition)